chart/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repositoryPrefix: "docker.io/flanksource" # Repository prefix, without trailing /
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: "flanksource-ui"
+fullnameOverride: "flanksource-ui"
+
+# Environment for deployment
+env:
+  - name: ORY_KRATOS_URL
```

Comment:
We can't default this

Reply:
ack, could you add this as secret to the to the aws-sandbox. Will update the chart with the URL from secret there.